lua/telescope/builtin/git.lua

```diff
@@ -297,6 +297,9 @@ git.branches = function(opts)
 
       map("i", "<c-d>", actions.git_delete_branch)
       map("n", "<c-d>", actions.git_delete_branch)
+
+      map("i", "<c-m>", actions.git_merge_branch)
+      map("n", "<c-m>", actions.git_merge_branch)
```

Comment:
`<c-m>` is not a good mapping. See `:help <cr>`. `<c-m>` and `<cr>` are the same meaning that we no longer can checkout a different branch with this PR

Reply:
Ooh this is why I had so much trouble using it at first! Damn I lost at least 15 minutes on this. 
Will update it rn